includes/wikia/api/ArticlesApiController.class.php

```diff
@@ -1021,6 +1023,12 @@ public function getAsJson() {
 
 		if ( $parsedArticle instanceof ParserOutput ) {
 			$articleContent = json_decode( $parsedArticle->getText() );
+
+			if ( !empty( $sectionsToGet ) ) {
```

Comment:
I usually feel like its clearer to test for the positive when an ```if``` statement has an ```else``` that you can flip around.